test/test_proxy_tensor.py

```diff
@@ -1999,7 +1999,6 @@ def f(t):
     xfail('geqrf', ''),  # aten.geqrf.default - couldn't find symbolic meta function/decomposition
     xfail('histogram', ''),  # Could not run 'aten::histogram.bin_ct' with arguments from the 'Meta' backend. This c...
     xfail('histogramdd', ''),  # aten._histogramdd_bin_edges.default - couldn't find symbolic meta function/decomposition
-    xfail('kthvalue', ''),  # aten.kthvalue.default - couldn't find symbolic meta function/decomposition
```

Comment:
Add `kthvalue` to out_symbolic_tensor_failures. You added a meta for one of the two kthvalue operators but not the other one. This xfail was blocking both of those from showing up in CI